locale/en_GB/LC_MESSAGES/djangojs.po

```diff
@@ -1217,30 +1278,40 @@ msgstr ""
 "content will only be loaded after you give your consent. Only then will "
 "YouTube set Cookies that collect information on user behaviour."
 
-#: dsgvo-video-embed/js/dsgvo-video-embed.js:15
+#: dsgvo-video-embed/js/dsgvo-video-embed.js:13
 msgid "More information can be found in the privacy policy of Google under: "
 msgstr "More information can be found in the privacy policy of Google under: "
 
-#: dsgvo-video-embed/js/dsgvo-video-embed.js:17
-#: dsgvo-video-embed/js/dsgvo-video-embed.js:22
+#: dsgvo-video-embed/js/dsgvo-video-embed.js:15
 msgid "Link to Video: "
 msgstr "Link to Video: "
 
-#: dsgvo-video-embed/js/dsgvo-video-embed.js:17
-#: dsgvo-video-embed/js/dsgvo-video-embed.js:22
+#: dsgvo-video-embed/js/dsgvo-video-embed.js:15
 msgid "Play video"
 msgstr "Play video"
 
-#: dsgvo-video-embed/js/dsgvo-video-embed.js:19
+#: dsgvo-video-embed/js/dsgvo-video-embed.js:16
+msgid "external video"
+msgstr "external video"
+
+#: dsgvo-video-embed/js/dsgvo-video-embed.js:17
 msgid ""
-"Here you can find a video from Vimeo. To protect your data the content will "
-"only be loaded after you give your consent. Only then will Vimeo set Cookies "
+"here you can find a video from vimeo. to protect your data the content will "
+"only be loaded after you give your consent. only then will vimeo set cookies "
```

Comment:
Shouldn't start with `Here`? And 2nd `only` after `.` needs to be title case too.

Reply:
Yes, the wording seems wrong but comes from https://github.com/liqd/dsgvo-video-embed, will make an issue there